cmd/lava/internal/help/helpdoc.go

```diff
@@ -12,6 +12,9 @@ var HelpLavaYAML = &base.Command{
 Each Lava project is defined by a configuration file (usually named
 lava.yaml) that defines the parameters of the security scan.
 
+The config file is a YAML document with support for environment variables
+substition with ${MYVAR} notation.
```

Comment:
I would be a bit more formal with the wording used in the definition.

```suggestion
A Lava configuration file is a YAML document that supports environment
variable substitution with ${ENVVAR_NAME} notation.
```

Reply:
done